dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -628,13 +683,16 @@ class Timeseries {
   }
 
   /// Adds a value to this timeseries.
-  void add(double value) {
+  void add(double value, {@required bool isWarmingUp}) {
     if (value < 0.0) {
       throw StateError(
         'Timeseries $name: negative metric values are not supported. Got: $value',
       );
     }
     _allValues.add(value);
+    if (useCustomWarmUp && isWarmingUp) {
+      _warmUpFrameCount ++;
```

Comment:
nit: we prefer `_warmUpFrameCount += 1` syntax.

Reply:
Done.